GetDKAN/dkan_sci_metadata
# Extended metadata form improvements

connects #8 
connects #9 

Issues:

- Form elements expand too wide when Science Purpose/Methods paragraph is added
- Sticky headers are unneccessary
- Draggable order option makes the form too busy & overly complicated

### QA Steps
- Create a dataset, add a paragraphs field
- Scroll up and confirm the header does not 'stick' to the top of the view port
- 'Add science purpose / methods'
- Confirm the paragraph form is the same width as the rest of the form
